src/arc.ts

```diff
@@ -49,6 +50,33 @@ export async function runArcDiff(fileName: string): Promise<ArcMessage[]> {
   return fileOutput;
 }
 
+export async function runArcDiff(flags?: string[]) {
+  const bin = config.pathToArc;
+  const command = [bin, "diff"];
+  const args = ["--config", "editor='code -w'"];
```

Comment:
This won’t work if it’s running in remote ssh will it?

Reply:
The remote ssh server extension appears to remap `code` to open on the remote client.

```
$ which code
/home/dierksen/.vscode-server/bin/86405ea23e3937316009fc27c9361deee66ffbf5/bin/code
```

Tested by connecting to devbox, and running `arc diff --config editor="code -w"`

Reply:
Nice! Thanks for checking :+1: